src/views/CAs/CAInspector.tsx

```diff
@@ -171,7 +171,10 @@ export const CAInspector: React.FC<Props> = ({ caName, engines }) => {
                                         )
                                     }
                                     {
-                                        ((caData.certificate.type !== "EXTERNAL" && caData.certificate.status === CertificateStatus.Revoked) || (caData.certificate.type === "EXTERNAL")) && (
+                                        (
+                                            (caData.certificate.type !== "EXTERNAL" && caData.certificate.status !== CertificateStatus.Active) ||
```

Comment:
Simplify the condition, it is  `(caData.certificate.type == "EXTERNAL" || caData.certificate.status !== CertificateStatus.Active)`?